bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/i18n/unifi.properties

```diff
@@ -164,3 +177,16 @@ error.thing.offline.configuration_error = You must choose a UniFi Controller for
 error.thing.poe.offline.configuration_error = The configuration parameter macAddress must be set and not be empty.
 error.thing.poe.offline.nodata_error = No data for the PoE port could be found in the UniFi API data. See TRACE log for actual API data.
 error.thing.site.offline.configuration_error = The configuration parameter sid must be set and not be empty.
+
+# actions
+
+actions.unifi.generateVouchers.label = Generate
+actions.unifi.generateVouchers.description = Generate guest voucher(s)
+action.unifi.revokeAllVouchers.label = Revoke All
```

Comment:
Typo:
```suggestion
actions.unifi.revokeAllVouchers.label = Revoke All
```

Reply:
Maybe they should all be singular, `action`?

Reply:
Settled on action everywhere.